src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,296 @@
+    description={`Follow PX Blue's guideline for form controls.`}
+     avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+/>
+<MaterialDesignDescription 
+    title={'Steppers'}
+    url={'/patterns/steppers'}
+    description={`Follow PX Blue's guideline for navigating between pages.`}
+     avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+/>
+
+Currently, PX Blue supports the following authentication workflows:
+
+-   Registration
+-   Join organization
+-   Login
+-   Reset password
+
+We will now discuss them one by one.
+
+## Registration
+
+Registration is the process of user account creation. Depends on your project setup, the user might need to request to join various organizations, which we will discuss later. Registration includes the following variations:
+
+**Self-Registration**
+
+Self-registration involves registering an account without an email invitation. The workflow involves the following screens:
+
+-   Enter email address
+-   Accept EULA
+-   Verify email
+-   Create password
+-   Add account details
+-   Success & welcome
+
+**Registration via Invitation**
+
+Sometimes the organization admin might email the user an invitation. The workflow involves the following screens:
+
+-   Click through an email invitation
+-   Accept EULA
+-   Create password
+-   Add account details
+-   Success & welcome
+
+### Screens
+
+Most steps involved in self-registration and registration via invitation are very similar. Use the following screens for the registration workflow where appropriate.
+
+<ImageGrid
+    images={[EnterEmailAdress, AcceptEULA, VerifyEmail, CreatePassword, AccountDetails, WelcomeOnboard]}
+    caption={[
+        'Screens shared by the registration workflows.',
+        '1: Enter email address',
+        '2: Accept EULA',
+        '3: Verify email',
+        '4: Create password',
+        '5: Add account details',
+        '6: Success & welcome',
+    ]}
+/>
+
+## Joining an Organization
+
+In addition to account creation, some applications may require the user to join groups and organizations. Joining an organization workflow includes the following variations:
+
+**Request Access**
+
+This occurs when a user has an account but need to access an invitation-only organization. The workflow involves the following screens:
+
+-   Enter organization code
+-   Success & welcome (awaiting approval)
+-   Receive an email invitation
+-   Success & welcome
+
+**Enter Passcode**
+
+This occurs when anyone with an organization passcode can be added to the organization. The workflow involves the following screens:
+
+-   Enter organization code
+-   Enter organization passcode
+-   Success & welcome
+
+**Join with Invitation**
+
+An organization admin can add an existing user from their end too. In case the user doesn't have an account, they will first do the registration -> registration via invitation, then proceed to this step. The workflow involves the following screens:
```

Comment:
```suggestion
Joining via invitation occurs when a system admin extends an invitation for a user to join a group. If the user does not yet have an account, they will first proceed through the invitation-based registration workflow followed by the screens for joining the organization. This workflow includes the following screens:
```

Reply:
Sometimes I feel like writing in kindergarten English bc I know Joe will always be backing me off